src/ts/badgeView.ts

```diff
@@ -41,9 +47,7 @@ export class BadgeView {
   <rect x="${this.numberWidth + this.stateWidth}" y="0" style="fill:#999"
         width="${this.badgeWidth}" height="${this.badgeHeight}" />
   <!-- TODO: labels -->
-  <image x="${this.numberWidth + this.stateWidth}" y="0"
-        width="${iconSize}" height="${iconSize}"
-        xlink:href="${this.issue.assignee.avatar_url}"></image>
+  ${icons}
```

Comment:
I think `icons` is necessary to be joined with empty string `''`. Or does this code output extra comma like following?

```html
<image .../image>,<image .../image>
```